api/v1alpha1/helmrepository_types.go

```diff
@@ -107,6 +107,16 @@ func HelmRepositoryReadyMessage(repository HelmRepository) string {
 	return ""
 }
 
+// GetArtifact returns the latest artifact from the source if present in the status sub-resource.
```

Comment:
I tried to wrap the comments at around 80 chars width (same for other types).

Reply:
Changed